README.md

```diff
@@ -4,6 +4,10 @@
 
 A low level Postgres client library for Haskell. Used at Hasura in various production projects.
 
+This library is no longer developed here, but remains in case it is used as a
+project dependency. All further updates can be found in the
+[graphql-engine](https://github.com/hasura/graphql-engine/tree/main/server/lib/pg-client-hs) repository.
```

Comment:
This link does not exist yet as the monorepo has not yet been copied to the public Github.